packages/odyssey-react-mui/src/theme/components.types.ts

```diff
@@ -29,6 +29,7 @@ declare module "@mui/material/Button" {
     primary: true;
     secondary: true;
     danger: true;
+    tertiary: true;
```

Comment:
When did we add a tertiary button?

Reply:
It's been in Figma for a while now, new with the Rebrand.